app/_hub/imperva/imp-appsec-connector/_index.md

```diff
@@ -0,0 +1,285 @@
+# Beneath the next --- use Markdown (redcarpet flavor) and HTML formatting only.
+#
+# The remainder of this file is for free-form description, instruction, and
+# reference matter.
+# If you include headers, your headers MUST start at Level 2 (parsing to
+# h2 tag in HTML). Heading Level 2 is represented by ## notation
+# preceding the header text. Subsequent headings,
+# if you choose to use them, must be properly nested (eg. heading level 2 may
+# be followed by another heading level 2, or by heading level 3, but must NOT be
+# followed by heading level 4)
+###############################################################################
+# BEGIN MARKDOWN CONTENT
+---
+## How it works
+The plugin sends a copy of API call requests/responses to the Imperva API receiver. The receiver service destination address and port are specified as config parameters. Additional parameters are used to control how the API captures are sent. 
+
+### How to install
+
+> If you are using Kong's [Kubernetes Ingress Controller](https://github.com/Kong/kubernetes-ingress-controller), the installation is slightly different. Review the [docs for Kubernetes Ingress](https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/setting-up-custom-plugins/)
+
+The `.rock` file is a self-contained package that can be installed locally or from a remote server.
+
+If the LuaRocks utility is installed in your system (this is likely the case if you used one of the official installation packages), you can install the 'rock' in your LuaRocks tree (a directory in which LuaRocks installs Lua modules).
+
+### Install the Imperva Plugin
+
+```shell
+ luarocks install imp-appsec-connector
+```
+
+### Update your loaded plugins list
+In your `kong.conf`, append `imp-appsec-connector` to the `plugins` field (or `custom_plugins` if old version of Kong). Make sure the field is not commented out.
+
+```yaml
+plugins = bundled,imp-appsec-connector         # Comma-separated list of plugins this node
+                                	       # should load. By default, only plugins
+                                               # bundled in official distributions are
+                                               # loaded via the `bundled` keyword.
+```
+
+
+### Restart Kong
+
+After LuaRocks is installed, restart Kong before enabling the plugin
+
+```shell
+kong restart
+```
+
+### Enable the Imperva Plugin 
+
+Please follow the example plugin configuration in the Configurtion Reference to enable the plugin on a service, on a route or globally. For your reference, an example
+of enabling the plugin on a service is provided below:
+
+```shell
+$ curl -X POST http://<kong-domain>:<kong-port>/services/<your-kong-service-id>/plugins/ \
+  --data "name=imp-appsec-connector" \
+  --data "config.destination_addr=<imp_apisec_controller_backend_domain_or_addr>" \
+  --data "config.destination_port=<destination_port_default_8080>" \
+  --data "config.connection_type=<tcp_or_http>"
+```
```

Comment:
I would remove this entire section and add the values here into `value_in_examples` fields in the list above. That's how they'll get generated into the examples section. Currently, this section is redundant.